src/web/lib/hakuneko/frontend@classic-light/app.html

```diff
@@ -61,12 +63,16 @@
             }
             hakuneko-mangas {
                 box-sizing: border-box;
+                resize: horizontal;
+                overflow: auto;
             }
             hakuneko-chapters {
                 box-sizing: border-box;
+                flex-grow: 1;
             }
             hakuneko-jobs {
                 flex: 0;
+                width: -webkit-fill-available;
```

Comment:
do not use browser specific css prefix, prefer standard fill-available

Reply:
![](https://i.vgy.me/aOg7RX.png)
invalid